### PR TITLE
Polished the Python Image Classifier Task Library API

### PR DESCRIPTION
**Pull request by kinaryml**

### CODE REVIEW

**Review by ziyeqinghan — APPROVED** [review]

**Review by ziyeqinghan — COMMENTED** [review]
Please add unittest for this.

**Comment by ziyeqinghan** [comment]
undo approval. Please add unittest first.

File: tensorflow_lite_support/python/test/task/vision/image_classifier_test.py [hunk 1]
```diff
@@ -128,6 +128,39 @@ def test_classify_model(self, model_file_type, max_results,
     self.assertDeepAlmostEqual(
         image_result_dict, expected_result_dict, places=5)
```
Comment (kinaryml) [inline tensorflow_lite_support/python/test/task/vision/image_classifier_test.py:130]: @ziyeqinghan I've added the unit test

File: tensorflow_lite_support/python/test/task/vision/image_classifier_test.py [hunk 2]
```diff
@@ -128,6 +128,39 @@ def test_classify_model(self, model_file_type, max_results,
...
+    classifier1 = _ImageClassifier.create_from_options(options1)
+    # Checks the same classifier object.
+    self.assertEqual(classifier1, classifier1)
+
+    base_options2 = _BaseOptions(
+      model_file=_ExternalFile(file_name=self.model_path))
+    options2 = _ImageClassifierOptions(base_options=base_options2)
+    classifier2 = _ImageClassifier.create_from_options(options2)
+    # Checks the classifiers with same file name.
+    self.assertEqual(classifier2, classifier2)
```
Comment (ziyeqinghan) [inline tensorflow_lite_support/python/test/task/vision/image_classifier_test.py:144]: classifier1

Comment (kinaryml) [reply tensorflow_lite_support/python/test/task/vision/image_classifier_test.py]: @ziyeqinghan Thanks for pointing it out! Fixed it

**Review by ziyeqinghan — APPROVED** [review]

**Pull request merged**